`Operator` defines `_flatten` and `_unflatten` methods (#4314)
PennyLaneAI/pennylane: doc/development/deprecations.rst

Pending deprecations
--------------------

> * The `RandomLayers.compute_decomposition` keyword argument `ratio_imprivitive` has been changed to `ratio_imprim` to

Reviewer: ```suggestion
* The `RandomLayers.compute_decomposition` keyword argument `ratio_imprivitive` will be changed to `ratio_imprim` to
```

Pending deprecations
--------------------

* The `RandomLayers.compute_decomposition` keyword argument `ratio_imprivitive` has been changed to `ratio_imprim` to
  match the call signature of the operation. 

> * The ``do_queue`` keyword argument in ``qml.operation.Operator`` is deprecated. This affects
>   all child classes, such as ``Operation``, ``Observable``, ``SymbolicOp`` and more. Instead of
>   setting ``do_queue=False``, use the ``qml.QueuingManager.stop_recording()`` context.
> 

Author: ```suggestion
```
